### PR TITLE
Shared containers

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: src/components/shared/InputFile/__tests__/container.spec.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    const frMock = {
+      readAsDataURL,
+      onloadend,
+    };
+    global.FileReader = jest.fn(() => frMock);
+
+    instance.handleChange(event);
+
+    it('reads file with correct size', () => {
+      expect(readAsDataURL).toBeCalled();
```
Comment (plzen) [inline src/components/shared/InputFile/__tests__/container.spec.js]: Don't needed here

File: src/components/shared/InputFile/container.js [hunk 2]
```diff
@@ -14,11 +14,13 @@ class InputFile extends Component {
     setFieldValue(name, fileRef);
     setFieldTouched(name);
 
+    /* istanbul ignore else */
```
Comment (plzen) [inline src/components/shared/InputFile/container.js]: Try to cover this

File: src/components/shared/InputFile/container.js [hunk 3]
```diff
@@ -14,11 +14,13 @@ class InputFile extends Component {
...
     if (fileRef) {
       reader.readAsDataURL(fileRef);
     }
 
     reader.onloadend = () => {
+      /* istanbul ignore else */
```
Comment (plzen) [inline src/components/shared/InputFile/container.js]: Try to cover this

File: yarn.lock [hunk 4]
```diff
@@ -3197,13 +3197,20 @@ convert-source-map@^0.3.3:
   resolved "https://registry.yarnpkg.com/convert-source-map/-/convert-source-map-0.3.5.tgz#f1d802950af7dd2631a1febe0596550c86ab3190"
   integrity sha1-8dgClQr33SYxof6+BZZVDIarMZA=
 
+<<<<<<< HEAD
```
Comment (plzen) [inline yarn.lock]: Rebase this file

File: src/components/shared/InputFile/__tests__/container.spec.js (hunk 1 above)
Comment (eugene-chr) [reply src/components/shared/InputFile/__tests__/container.spec.js]: done

File: yarn.lock (hunk 4 above)
Comment (eugene-chr) [reply yarn.lock]: done

File: src/components/shared/InputFile/container.js (hunk 2 above)
Comment (eugene-chr) [reply src/components/shared/InputFile/container.js]: done

File: src/components/shared/InputFile/container.js (hunk 3 above)
Comment (eugene-chr) [reply src/components/shared/InputFile/container.js]: done

**Pull request merged**